src/FilterAdder.js

```diff
@@ -2,21 +2,13 @@ import _ from 'lodash/fp'
 import React from 'react'
 import { observer } from 'mobx-react'
 import InjectTreeNode from './utils/injectTreeNode'
-import { DefaultNodeProps } from './utils/schema'
+import { newNodeFromField } from './utils/search'
 
 export let fieldsToOptions = _.map(x => ({ value: x.field, ...x }))
 
 let getGroupFields = node => _.map('field', _.getOr([], 'children', node))
 
-let FilterAdder = ({
-  tree,
-  node,
-  path,
-  fields,
-  Picker,
-  uniqueFields,
-  defaultNodeProps = DefaultNodeProps,
```

Comment:
@daedalus28 Do we want this? Nothing uses it.

Reply:
also even if it does not, contexture-react shouldn't dictate that's the only way to add default node props

Reply:
> contexture-react shouldn't dictate that's the only way to add default node props

Adding it back to merge this PR, but this flexibility is only supported here. We just go with a vanilla `defaultNodeProps` everywhere else

Reply:
> contexture-react shouldn't dictate that's the only way to add default node props

Why here and not in all the other places we're using it? This is the only place where we offer overriding it and we're not using it anyway.

Reply:
ok, we can always add this back later if need be then